### ISSUE TITLE
Travis CI:  failure about mysql-8.0 

### ISSUE THREAD

**Issue by kenhys** [mroonga/mroonga#256]
See https://travis-ci.org/mroonga/mroonga/jobs/568652433 Travis CI build logs.

FYI: This failure is caused by already known bug:
https://bugs.mysql.com/bug.php?id=96266

**Comment by kenhys** [comment]
In debian/rules, it seems that `override_dh_auto_configure` will not accept additional options.

**Comment by kenhys** [comment]
-DDOWNLOAD_BOOST=1 should be used in debian/rules

**Comment by kenhys** [comment]
It seems that rewriting debian/rules is effective.
CI issue will be fixed in PR #257

**Comment by kenhys** [comment]
It seems that CI for MySQL 8.0 is running expectedly.
https://travis-ci.org/kenhys/mroonga/jobs/568740917

**Comment by kenhys** [comment]
https://bugs.mysql.com/bug.php?id=96266 will be fixed in 8.0.18. (not released yet)

**Comment by kenhys** [comment]
```
Warnings:
Warning	1681	Integer display width is deprecated and will be removed in a future release.
```
will be fixed by https://github.com/mroonga/mroonga/pull/258

**Comment by komainu8** [comment]
Fixed by #257, #258, and #259.
So, we close this issue.
